Core/PixelEvent.swift

```diff
@@ -1367,6 +1370,8 @@ extension Pixel.Event {
             // MARK: debug pixels
             
         case .dbCrashDetected: return "m_d_crash"
+        case .crashReportCRCIDMissing: return "crashreporting_crcid-missing"
+        case .crashReportingSubmissionFailed: return "crashreporting_submission-failed"
```

Comment:
These will need to have the `m_` prefix at the start, otherwise they won't get processed correctly by the pixel endpoint. Something like `m_crashreporting_crcid-missing` is fine. This isn't needed on macOS since the prefixing is handled automatically, but iOS is a bit behind 🙂